README.md

```diff
@@ -29,18 +29,31 @@ See https://interledgerjs.github.io/ilp-protocol-psk2
 Uses [`createReceiver`](https://interledgerjs.github.io/ilp-protocol-psk2/modules/_receiver_.html#createreceiver) and [`Receiver.generateAddressAndSecret`](https://interledgerjs.github.io/ilp-protocol-psk2/classes/_receiver_.receiver.html#generateaddressandsecret).
 
 ```js
-const { createReceive } = require('ilp-protocol-psk2')
-const receiver = await createReceiver({
-  plugin: myLedgerPlugin,
-  paymentHandler: async (params) => {
-    // Accept all incoming payments
-    const result = await params.accept()
-    console.log('Got payment for:', result.receivedAmount)
-  }
-})
-
-const { destinationAccount, sharedSecret } = receiver.generateAddressAndSecret()
-// Give these two values to a sender to enable them to send payments to this Receiver
+const crypto = require('crypto')
+const base64url = require('base64url')
+const IlpPluginBtp = require('ilp-plugin-btp')
+const { createReceiver } = require('.')
+
+function tokenToAccount (token) {
+  return base64url(crypto.createHash('sha256').update(token).digest('sha256'))
+}
+
+;(async () => {
+  const receiverToken = base64url(crypto.randomBytes(32))
+  const receiverAccount = tokenToAccount(receiverToken)
+  const receiver = await createReceiver({
+    plugin: new IlpPluginBtp({ server: `btp+wss://${receiverAccount}:${receiverToken}@amundsen.ilpdemo.org:1801` }),
+    paymentHandler: async (params) => {
+      // Accept all incoming payments
+      const result = await params.accept()
+      console.log('Got payment for:', result.receivedAmount)
+    }
+  })
+
+  const { destinationAccount, sharedSecret } = receiver.generateAddressAndSecret()
+  console.log({ destinationAccount, sharedSecret })
+  // Give these two values to a sender to enable them to send payments to this Receiver
+})()
```

Comment:
This should be run with `.catch((err) => console.log(err))` just in case something goes wrong